saleor/graphql/app/tests/queries/test_app.py

```diff
@@ -131,3 +131,92 @@ def test_app_with_access_to_resources(
         query, permissions=[permission_manage_orders]
     )
     get_graphql_content(response)
+
+
+def test_own_app_without_id(
+    app_api_client,
+    app,
+    permission_manage_orders,
+    order_with_lines,
+):
+    webhook = Webhook.objects.create(
+        name="Simple webhook", app=app, target_url="http://www.example.com/test"
+    )
+    webhook.events.create(event_type=WebhookEventType.ORDER_CREATED)
+
+    response = app_api_client.post_graphql(
+        QUERY_APP,
+    )
+    content = get_graphql_content(response)
+
+    tokens = app.tokens.all()
+    app_data = content["data"]["app"]
+    tokens_data = app_data["tokens"]
+    assert tokens.count() == 1
+    assert tokens_data[0]["authToken"] == tokens.first().auth_token[-4:]
+
+    assert app_data["isActive"] == app.is_active
+    assert len(app_data["webhooks"]) == 1
+    assert app_data["webhooks"][0]["name"] == webhook.name
+    assert app_data["type"] == app.type.upper()
+    assert app_data["aboutApp"] == app.about_app
+    assert app_data["dataPrivacy"] == app.data_privacy
+    assert app_data["dataPrivacyUrl"] == app.data_privacy_url
+    assert app_data["homepageUrl"] == app.homepage_url
+    assert app_data["supportUrl"] == app.support_url
+    assert app_data["configurationUrl"] == app.configuration_url
+    assert app_data["appUrl"] == app.app_url
+
+
+def test_app_query_without_permission(
+    app_api_client,
+    app,
+    order_with_lines,
+):
+    id = graphene.Node.to_global_id("App", app.id)
+    variables = {"id": id}
+    response = app_api_client.post_graphql(
+        QUERY_APP,
+        variables,
+    )
+    assert_no_permission(response)
+
+
+def test_app_query_with_permission(
```

Comment:
We should add a similar test executed as staff.

Reply:
we already have test fot that case `test_app_query`